bazarr/api/system/status.py

```diff
@@ -15,13 +15,21 @@
 class SystemStatus(Resource):
     @authenticate
     def get(self):
+        package_version = ''
+        if  'BAZARR_PACKAGE_VERSION' in os.environ:
+            package_version = os.environ['BAZARR_PACKAGE_VERSION']
+        if 'BAZARR_PACKAGE_AUTHOR' in os.environ and os.environ['BAZARR_PACKAGE_AUTHOR'] != '':
```

Comment:
Why not just use `if os.environ.get('BAZARR_PACKAGE_AUTHOR'):`
Result will be same because get method return `None` i key is not in dict
and also empty string return `False` in condition

Reply:
Comparing against `None` should be done with `is not None`.

In the end it doesn't really matter, outcome is the same, it's just a matter of preference.
Let's have @morpheus65535 decide